### DISCUSSION TITLE
-max-redirects & -disable-redirects

### DISCUSSION THREAD

**Discussion by Techbrunch** [projectdiscovery/nuclei#2593] [Ideas]
I tried to use `-max-redirects 0` before using `-disable-redirects` to disable redirects but I was suprised to see that `-max-redirects 0` is not equivalent to `-disable-redirects`.

Don't you think it should be the case ?

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
